--- jsoncpp-1.9.6/jsoncpp-namespaced-targets.cmake.orig	2024-09-10 05:18:29.000000000 +0200
+++ jsoncpp-1.9.6/jsoncpp-namespaced-targets.cmake	2025-08-27 20:27:33.362448908 +0200
@@ -1,9 +1,9 @@
 if (NOT TARGET JsonCpp::JsonCpp)
-    if (TARGET jsoncpp_static)
-        add_library(JsonCpp::JsonCpp INTERFACE IMPORTED)
-        set_target_properties(JsonCpp::JsonCpp PROPERTIES INTERFACE_LINK_LIBRARIES "jsoncpp_static")
-    elseif (TARGET jsoncpp_lib)
+    if (TARGET jsoncpp_lib)
         add_library(JsonCpp::JsonCpp INTERFACE IMPORTED)
         set_target_properties(JsonCpp::JsonCpp PROPERTIES INTERFACE_LINK_LIBRARIES "jsoncpp_lib")
+    elseif (TARGET jsoncpp_static)
+        add_library(JsonCpp::JsonCpp INTERFACE IMPORTED)
+        set_target_properties(JsonCpp::JsonCpp PROPERTIES INTERFACE_LINK_LIBRARIES "jsoncpp_static")
     endif ()
 endif ()
